mocks/DUMMY_DATA.tsx

```diff
@@ -0,0 +1,180 @@
+export const perks = [
```

Comment:
lets organize this file in case someone will want to use it
low priority

Reply:
without mistaken, we are all dynamic now. 
do you still want to keep it?